datastore/builtin/build_test.go

```diff
@@ -0,0 +1,207 @@
+package builtin
+
+import (
+	"github.com/drone/drone/common"
+	"github.com/franela/goblin"
+	"testing"
+)
+
+func TestBuildstore(t *testing.T) {
+	db := mustConnectTest()
+	rs := NewRepostore(db)
+	bs := NewBuildstore(db)
+	cs := NewCommitstore(db)
+	defer db.Close()
+
+	g := goblin.Goblin(t)
+	g.Describe("Buildstore", func() {
+
+		// before each test we purge the package table data from the database.
+		g.BeforeEach(func() {
+			db.Exec("DELETE FROM blobs")
+			db.Exec("DELETE FROM builds")
+			db.Exec("DELETE FROM commits")
+			db.Exec("DELETE FROM repos")
+			db.Exec("DELETE FROM stars")
+			db.Exec("DELETE FROM tasks")
+			db.Exec("DELETE FROM tokens")
+			db.Exec("DELETE FROM users")
+		})
```

Comment:
we probably don't need to delete data from all tables, since we are only modifying the `builds` table